build.gradle

```diff
@@ -27,7 +31,7 @@ allprojects {
     repositories {
         mavenLocal()
         maven {
-            url "https://development.galasa.dev/main/maven-repo/wrapping/"
+            url "https://maven.pkg.github.com/jaydee029/wrapping/"
```

Comment:
Of course we'll need to change this to galasa-dev when ready but fine for now while testing